pylint/checkers/variables.py

```diff
@@ -1927,22 +1927,28 @@ def _check_consumer(
 
                 # Skip postponed evaluation of annotations
                 # and unevaluated annotations inside a function body
-                if not (
-                    self._postponed_evaluation_enabled
-                    and (
+                if (
```

Comment:
The formatting start being a problem here, especially on mobile![Screenshot_20250810-063413.jpg](https://github.com/user-attachments/assets/2dcb6026-8c93-4112-9e12-ec576d25b501)

Reply:
Yeah, saw that as well. I just added one line `and not isinstance(stmt, nodes.TypeAlias)` but that triggered a whole reformatting by black. Ruff also doesn't make it any easier with [`RUF021`](https://docs.astral.sh/ruff/rules/parenthesize-chained-operators/). Believe I found a somewhat acceptable solution by changing `not and` to `not (... or ...)` and disabling `RUF021` in a few places.